### ISSUE TITLE
Choose a license (currently it's "UNLICENSED")

### ISSUE THREAD

**Issue by laurence-myers** [typepoint/typepoint#9]
Currently package.json specifies the license as "UNLICENSED". This means no-one has a license to use this lib.

**Comment by codeandcats** [comment]
Yeah, that's a tad more restrictive than I intended hehe.

I was a little unimpressed after someone forked one of my other repos, made performance improvements, but rather than submitting a PR, they republished as their own library. I think I used the MIT license for that project, so they were well within their rights to have done that - just as I would have been well within my rights to take their changes and push them back into my library, but since it was in C# and don't use DotNet for anything anymore I didn't bother.

Since then I've used UNLICENSED for some libraries that I don't want people to simply fork and rebadge, but you're right it's a little restrictive. I had trouble finding a ready-made license that basically lets users use the library, submit improvements, etc but not republish and redistribute their own forked version (likely because its a little counter open-source) so I might need to just write a custom license agreement.

I'm open to your thoughts and feelings though if you have strong thoughts about it.
